src/adaptations/device-layer/include/Weave/DeviceLayer/EFR32/gatt.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file needs an OpenWeave licence header.

Reply:
Done